ci-operator/config/openshift/knative-serving/openshift-knative-serving-release-v0.11.1__4.3.yaml

```diff
@@ -1,207 +0,0 @@
-base_images:
```

Comment:
You're keeping serverless release-1.4 so perhaps we should leave also this knative serving release v0.11.1. Or do you want to run the test suite as part of the serverless-operator repo?

Reply:
Good point, I fumbled the version correlation.